oada-core/http-handler/oada-srvc-http-handler/config.ts

```diff
@@ -0,0 +1,5 @@
+import libConfig from '../../libs/oada-lib-config'
+// @ts-ignore
+import defaults from './config.defaults'
```

Comment:
Not that this should hold up this PR, but we really should rethink config. Its not compatible in any nice way with Typescript (or not that I could find) and doesn't do anything for us that something like `dotenv` doesn't already do. As far as I could tell, we only ever use ENV vars to configure our microservices.

Reply:
I agree with redoing the config nonsense at some point.